pyproject.toml

```diff
@@ -1,20 +1,20 @@
 [tool.poetry]
-name = "python-tahoma-api"
-version = "0.1.0"
-description = "Python wrapper to interact with internal Somfy TaHoma API"
+name = "pyhoma"
+version = "0.2.0"
+description = "Async Python wrapper to interact with internal Somfy TaHoma API"
 authors = ["Mick Vleeshouwer", "Vincent Le Bourlot", "Thibaut Etienne"]
 license = "MIT"
 readme = "README.md"
 homepage = "https://github.com/iMicknl/python-tahoma-api"
 repository = "https://github.com/iMicknl/python-tahoma-api"
 packages = [
-    { include = "tahoma_api" },
-    { include = "tahoma_api/**/*.py" },
+    { include = "tahoma" },
```

Comment:
We need this also? The next line is not enough?

Reply:
By the way, should we rename this package to pyhoma now or for the next release?

Reply:
@tetienne I guess even the top one would be enough. I copied it from here: https://python-poetry.org/docs/pyproject/#packages. But I missed that they used `my_package` and `other_package`.

Reply:
@tetienne I would prefer to change everything directly, before the first publish. We just need to hotfix the other component, but wouldn't be a big hassle.

Reply:
I think you can even remove the packages fields. Look at https://github.com/tetienne/somfy-open-api/blob/master/pyproject.toml. Don't use it and my package is ok: https://pypi.org/project/pymfy/#files

Reply:
OK, I let do it.